bcipy/task/paradigm/rsvp/copy_phrase.py

```diff
@@ -420,6 +425,14 @@ def execute(self) -> str:
             self.parameters['preview_inquiry_progress_method'],
             self.trigger_handler.file_path).as_dict()
         self.write_session_data()
+
+        # Evidence is not recorded in the session when using fake decisions.
+        if self.parameters['summarize_session'] and self.session.has_evidence():
+            session_db(data_dir=self.file_save,
```

Comment:
Yes, good point. I refactored the session helper so that the functionality for generating the evidence records is no longer coupled with the code for creating the database. Now we can generate the spreadsheet without first creating a db.